docs/_workshops/object-detection/Annotations to Colab.md

```diff
@@ -0,0 +1,27 @@
+
+---
+title: Training a Model
+order: 6
+redirect_from:
+  - /object-detection/cli/6.html
+---
+After we have collected and labeled our first round of images, we are ready to start training our model! 
+
+## Prepare to Export to Colab
+To train our model we need to install the Cloud Annotation CLI.
+
+#### Click Train in Colab
+There's a green button on the top right of your screen that says "Train in Colab". Click that to get started.
+[!Train Button](docs/_workshops/object-detection/assets/colab_training_button.png)
+
+#### Copy Credentials
+A popup will appear with necessary credentials for access to your bucket. Click to copy those.
```

Comment:
give a summary on what the credentials are being using for

and same note as above on back ticks: 
```
Click `copy` to copy your credentials
```